_posts/2023-04-30-writing-code-with-llm-dont-forget-kani.md

```diff
@@ -0,0 +1,306 @@
+---
+layout: post
+title: "Writing Code with LLM? Don't Forget Kani"
```

Comment:
option 1: LLMs for code? Don't forget Kani... 
option 2: Kani to the rescue
maybe others have more ideas.

Reply:
Option - Writing Code with ChatGPT? Improve it with Kani.

Since we're strictly talking about ChatGPT and people are more familiar with ChatGPT as a term than LLMs. 

Don't forget Kani is a bit vague imo.

Reply:
I totally agree with @jaisnan. People are way more familiar with ChatGPT.